pkg/api/handlers/compat/images_push.go

```diff
@@ -156,6 +156,7 @@ loop: // break out of for/select infinite loop
 					Current: int64(e.Offset),
 					Total:   e.Artifact.Size,
 				}
+				report.ProgressMessage = report.Progress.String()
```

Comment:
It is not related to https://github.com/containers/podman/pull/14886. It is just miss in compat with docker found here: https://github.com/containers/podman/pull/14886#issuecomment-1181423589